action.yml

```diff
@@ -72,19 +72,19 @@ runs:
     - name: Check if .exclude-${{ steps.vars.outputs.distribution }} is present in ${{ steps.vars.outputs.dockerfile_path }} directory
       id: check_exclude_file
       # https://github.com/marketplace/actions/file-existence
-      uses: andstor/file-existence-action@v1
+      uses: andstor/file-existence-action@v2
```

Comment:
Only for my understanding. What is difference between `v1` and `v2`?

Reply:
Here are release notes: https://github.com/andstor/file-existence-action/releases

The main reason we are bumping is that v1 contains deprecated set-output, which spams us with warnings in every run of build-and-push action.